samples/cellular/nrf_cloud_multi_service/child_image/mcuboot/boards/nrf9160dk_nrf9160.conf

```diff
@@ -5,6 +5,7 @@
 #
 
 CONFIG_SPI=y
+# Required by QSPI
```

Comment:
I realize this is an old comment and not something you wrote, but, the 9160 does not have QSPI hardware, so the comment should be `# Required by SPI`.